devel/openssl/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           muniversal 1.0
 
 name                openssl
 epoch               1
-version             1.0.2p
+version             1.0.2q
 
 # Please revbump these ports when updating OpenSSL.
 #  - freeradius (#43461)
```

Comment:
Please also add a revbump for `freeradius` and `openssh` as the comment explains.